packages/sanity/src/core/form/types/blockProps.ts

```diff
@@ -444,5 +444,10 @@ export interface BlockProps {
  */
 export interface PortableTextPluginsProps {
   renderDefault: (props: PortableTextPluginsProps) => React.JSX.Element
-  plugins: {markdown: {config: MarkdownPluginConfig}}
+  plugins: {markdown: {config: MarkdownConfig}}
+}
+
+export type MarkdownConfig = Omit<MarkdownShortcutsPluginProps, 'unorderedList' | 'orderedList'> & {
```

Comment:
Part of the reason why the CI is failing is because this is not marked as  @alpha, @beta, @public, or @internal and we're exporting it :) 
Try adding one of the labels in a comment as you can see in other places  and I think CI will start passing!